setup.py

```diff
@@ -53,7 +59,12 @@
     ],
     license="GPL v3.0",
     url="https://unitary.fund",
-    python_requires='>=3.7',
+    project_urls={
+        "Bug Tracker": "https://github.com/unitaryfund/mitiq/issues/",
+        "Documentation": "https://mitiq.readthedocs.io/en/latest/",
```

Comment:
```suggestion
        "Documentation": "https://mitiq.readthedocs.io/en/stable/",
```
Maybe the `stable` docs is more appropriate if this link is shown in PyPI releases.